### PR TITLE
@grafana/runtime: expose config and loadPluginCss

### PR DESCRIPTION
**Pull request by ryantxu**
expose config and loadPluginCss via @grafana/runtime

### CODE REVIEW

**Comment by ryantxu** [comment]
@dprokop -- any thoughts on how to get around this?
image

or alternative approach?

**Comment by dprokop** [comment]
I would try add @types/system js and simply use `import System from 'systemjs';` instead of direct import from dist file. This should help resolve this problem.

File: packages/grafana-runtime/src/config.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+export interface BuildInfo {
+  version: string;
+  commit: string;
+  isEnterprise: boolean;
+  env: string;
+  latestVersion: string;
+  hasUpdate: boolean;
+}
+
+export class Settings {
```
Comment (torkelo) [inline packages/grafana-runtime/src/config.ts]: Would be good to take the time to change this type name to something config (as the file is called config). Maybe AppBootConfig or just GrafanaAppConfig or along that vein.

**Review by torkelo — APPROVED** [review]
just one minor comment otherwise good

File: packages/grafana-runtime/src/config.ts (hunk 1 above)
Comment (ryantxu) [reply packages/grafana-runtime/src/config.ts]: I renamed it GrafanaBootConfig :)

**Pull request merged**